label_studio/users/models.py

```diff
@@ -190,6 +204,11 @@ def reset_token(self):
 
     def get_initials(self):
         initials = '?'
+
+        if flag_set('fflag_feat_all_optic_114_soft_delete_for_churned_employees', user='auto'):
```

Comment:
shouldn't we pass `self` as user? otherwise this won't work if used in redis queues for example